core/tee/tee_rpmb_fs.c

```diff
@@ -1264,18 +1264,94 @@ static TEE_Result tee_rpmb_read(uint16_t dev_id, uint32_t addr, uint8_t *data,
 	return res;
 }
 
+static TEE_Result write_req(uint16_t dev_id, uint16_t blk_idx,
+			    const void *data_blks, uint16_t blkcnt,
+			    const uint8_t *fek, const TEE_UUID *uuid,
+			    struct tee_rpmb_mem *mem, void  *req, void *resp)
+{
+	TEE_Result res = TEE_SUCCESS;
+	uint8_t hmac[RPMB_KEY_MAC_SIZE] = { 0 };
```

Comment:
it's a u8 array, so `0` is ok.

Reply:
It is certainly valid, but our coding rules recommend `{ };` always don't they? (or do we have exceptions?)

Reply:
Coding rules states:
> * Scalars (and types like time_t which are standardized as scalars) are initialized with 0, unless another value makes more sense.
> * (...)
> * For the rest (...) we initialize structs with { } in order to avoid the more clumsy memset() procedure. (...)

Nothing specific about arrays so here both `{ }` and `{ 0 }` are ok I think.

Reply:
OK, I'll change to `{ }` to keep things simpler.